### DISCUSSION TITLE
Howto import IMAP emails

### DISCUSSION THREAD

**Discussion by MarcS1975** [docker-mailserver/docker-mailserver#4171] [Q&A] [answered]
Hello community. I have been running DMS for a while now for several live domains. Email runs well.
Now I have a legacy IMAP account which is no longer connected to any domain. I only have the (full) backup of all IMAP files. 
How can I import this data into DMS? I dont need the data to be able to send/receive. I'd only like to read the imported emails.

I am attaching a list of the folders I have for import.

[imap folders import.txt](https://github.com/user-attachments/files/16895788/imap.folders.import.txt)

Any tips?
(most grateful)

**Comment by casperklein** [comment, marked as answer]
You could try:

1. Create a new user acccount
2. Copy over those files and directories to `maildata/$domain/$new-user-account`

**Comment by MarcS1975** [comment]
Many thanks! The simple copy seems to have worked.
Is there no index re-build that DSM (dovecot?) needs to run?

**Reply by casperklein** [reply]
No. If you use the [maildir](https://en.wikipedia.org/wiki/Maildir) format, which is the default in DMS, then there is no indexing or something. It's just plain (email) files.

**Comment by MarcS1975** [comment]
Thank you very much. The result is amazing! All emails visible.
